checker/src/test/java/org/checkerframework/checker/test/junit/InterningWarnRedundantAnnosTest.java

```diff
@@ -0,0 +1,29 @@
+package org.checkerframework.checker.test.junit;
+
+import org.checkerframework.framework.test.CheckerFrameworkPerDirectoryTest;
+import org.junit.runners.Parameterized;
+
+import java.io.File;
+import java.util.List;
+
+/** JUnit tests for the Interning checker when AwarnRedundantAnnotations is used. */
+public class InterningWarnRedundantAnnosTest extends CheckerFrameworkPerDirectoryTest {
+
+    /**
+     * Create a InterningWarnRedundantAnnosTest.
+     *
+     * @param testFiles the files containing test code, which will be type-checked
+     */
+    public InterningWarnRedundantAnnosTest(List<File> testFiles) {
+        super(
+                testFiles,
+                org.checkerframework.checker.interning.InterningChecker.class,
+                "interning-warnredundantannos",
+                "-AwarnRedundantAnnotations");
+    }
+
+    @Parameterized.Parameters
+    public static String[] getTestDirs() {
+        return new String[] {"interning-warnredundantannos"};
```

Comment:
Throughout, can you expand `annos` to `annotations`, to be uniform with the existing `nullness-warnredundantannotations` test names?

Reply:
Fixed!